### PR TITLE
update padding for bars where y value is negative

### LINKED ISSUE #124: label padding is off when values are negative
_From @boygirl on December 14, 2015 5:12_

labels are too close to bars for negative bars both horizontal and vertical

_Copied from original issue: FormidableLabs/victory-bar#63_

### PR DESCRIPTION
**Pull request by ebrillhart**
fixes #124 

@boygirl

### CODE REVIEW

File: src/components/victory-bar/helper-methods.js [hunk 1]
```diff
@@ -111,7 +111,10 @@ export default {
       );
 
       const text = this.getLabel(modifiedProps, datum, index);
-      const labelStyle = this.getLabelStyle(style.labels, datum);
+      const labelPadding = style.labels.padding;
+      const labelStyleObj = datum.y < 0 ? Object.assign({}, style.labels,
+        {padding: labelPadding - 7}) : style.labels;
```
Comment (boygirl) [inline src/components/victory-bar/helper-methods.js]: Magic number 7? Can you check that this is correct with different font sized and multi-line labels.

Comment (ebrillhart) [inline src/components/victory-bar/helper-methods.js]: yep - tried multi-line, a few different font sizes, and throwing on a theme, and it worked well across all of those!

Comment (boygirl) [inline src/components/victory-bar/helper-methods.js]: I believe you, but I would still be more comfortable if this number were expressed in terms of fontSize or padding or something less magical

Comment (ebrillhart) [inline src/components/victory-bar/helper-methods.js]: fair enough!

File: src/components/victory-bar/helper-methods.js [hunk 2]
```diff
@@ -113,7 +113,7 @@ export default {
       const text = this.getLabel(modifiedProps, datum, index);
       const labelPadding = style.labels.padding;
       const labelStyleObj = datum.y < 0 ? Object.assign({}, style.labels,
-        {padding: labelPadding - 7}) : style.labels;
+        {padding: labelPadding * -Math.abs(labelPadding / 2.5)}) : style.labels;
```
Comment (boygirl) [inline src/components/victory-bar/helper-methods.js]: :+1:

**Comment by boygirl** [comment]
Approved!

**Pull request merged**